statusfy/overlays/cnv-prod/route.enc.yaml

```diff
@@ -0,0 +1,178 @@
+kind: Route
```

Comment:
This is not encrypted @goern , our root .sops file isn't updated for encrypting tls files. 
we just kept the sops file for encrypting tls files in respective repos https://github.com/thoth-station/thoth-application/blob/master/user-api/.sops.yaml

two solution
1. copy the sops file https://github.com/thoth-station/thoth-application/blob/master/user-api/.sops.yaml to the statusfy dir. Then do the `sops -e` from within the dir for the routes file
2. we can update the sops file at the root , for encrypting the tls as well, so that we dont have to encrypting from within a dir. 

However we definitely have to create new certs as these are exposed.